test/persistence/persistence.jl

```diff
@@ -1,7 +1,4 @@
 @testset "Persistence" begin
-    @testset "Errors" begin
```

Comment:
@sbromberger the test removed here feels like a "comfort test" more than anything, it's literally just the function behavior